script/QA/QA_02.txt

```diff
@@ -97,7 +97,7 @@ We'll explain those cases when the time comes.
 [sha:11100ece80f4d0c7dd8198ad8798434dc5531f1a]{
 -- Page 39, Offset 42840.
 -- 今回はよくある『即死系デッドエンド』です。
- This time,%{n}we have the common 'instant death dead end'.
+ This time,%{n}we have the common "instant death dead end".
```

Comment:
Should still be single quotes I think?

Reply:
I thought it's supposed to be double quotations and @Avuxo agreed so which is it?

Reply:
although these are in-text quotes, they're specifically _not_ wrapped in any form of quotation. That's why I said double quotes. If Ross thinks we should take the implicit quotes as a reason to use single quotes in QA that's fine.

Reply:
Yeah I feel like it's standard to use single quotes for non-speech 'things' like titles or terminology

Reply:
I'm fine to have it be reverted